### PR TITLE
CSS Styling 

### PR DESCRIPTION
**Pull request by kritiagrawal1997**
Aligned all components
Applied Grid 
Applied Flex inside each employee contribution component
Applied Media Query

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.org/XInterns/Contributions-FE.svg?branch=master)](https://travis-ci.org/XInterns/Contributions-FE)
-
```
Comment (Deepankar01) [inline README.md:2]: please add this don't remove this

File: public/index.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```
Comment (Deepankar01) [inline public/index.html:13]: where are you using this.. ?

File: src/Components/Box/index.css [hunk 3]
```diff
@@ -1,12 +1,45 @@
 .Rectangle-2
```
Comment (Deepankar01) [inline src/Components/Box/index.css]: give an appropriate name for the same

File: src/Components/Box/index.css [hunk 4]
```diff
@@ -1,12 +1,45 @@
...
+            }
+
+      .row2{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        /* background-color:burlywood; */
+      }
+
 .Rectangle-2:hover
```
Comment (Deepankar01) [inline src/Components/Box/index.css]: appropriate name

File: src/Components/Box/index.css [hunk 5]
```diff
@@ -34,23 +67,4 @@
...
-    {
-      border:2px;
-      border-color: black;
-      width:100%;
-      margin-top: 4%;
-    }
-}
-
-
 @import url('https://fonts.googleapis.com/css?family=Muli:400');
```
Comment (Deepankar01) [inline src/Components/Box/index.css]: put this in the top if at all using this

File: src/Components/Box/index.js [hunk 6]
```diff
@@ -10,11 +10,26 @@ export default class Box extends Component {
...
         <div className="Rectangle-2">
-          <Picture pic={(this.props.it).name} />
-          <Name name={(this.props.it).name} />
-          <Date date={(this.props.it).height} />
-          <Contribution content={(this.props.it).mass+"sadjkandkjasndjaskndkasjndaskjjd bnkjasnbdkjasdkjasbdkaj sbddkjasjbdkja sjbdkjasbdasskj jdbaskjjdbsakj dbaskjbdkjas jbdkjasbdkja sjbdkjasbdkasd bsaskjjdbsak jdbkjasdjasjdk jasjdbasbdkja sbdkjasbdkjas bdjkasdb b"} />
+         
+          <div className="row1">
+
+              <div className="col1">
+                  <Picture pic={(this.props.it).contributor_name} />
```
Comment (Deepankar01) [inline src/Components/Box/index.js]: good

File: src/Components/Box/index.js [hunk 7]
```diff
@@ -10,11 +10,26 @@ export default class Box extends Component {
...
+                  <Name name={(this.props.it).contributor_name} />
```
Comment (Deepankar01) [inline src/Components/Box/index.js]: why brackets.. ? (this.props.it).contributor_name can be this.props.it.contributor_name

File: src/Components/Contribution/index.css [hunk 8]
```diff
@@ -15,9 +14,3 @@
     font-family: 'Muli', sans-serif;
```
Comment (Deepankar01) [inline src/Components/Contribution/index.css]: if you are using `Muli` font put it in one place

File: src/Components/Date/index.css [hunk 9]
```diff
@@ -1,11 +1,10 @@
...
-    margin-top:-4%;
-    margin-left: 75%;
+   
     opacity: 0.5; 
     font-size: 12px;
-    line-height: 1.85;
+    line-height: 9px;
     letter-spacing: 0.3px;
     color: #4a4a4a;
     font-family: 'Muli', sans-serif;
```
Comment (Deepankar01) [inline src/Components/Date/index.css]: again here

File: src/Components/Name/index.css [hunk 10]
```diff
@@ -1,11 +1,12 @@
...
     font-size: 14px;
     font-weight: 900;
     letter-spacing: 0.4px;
     color: #4a4a4a;
-    line-height: 9px;
     font-family: 'Muli', sans-serif;
-    
-  }
\ No newline at end of file
+    /* background-color:orangered; */
```
Comment (Deepankar01) [inline src/Components/Name/index.css]: remove dead code

File: src/Components/Rectangle/index.css [hunk 11]
```diff
@@ -28,10 +28,10 @@
   
 
 
-
+/*
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.css]: remove dead code

File: src/Components/Rectangle/index.js [hunk 12]
```diff
@@ -31,7 +31,7 @@ class Rectangle extends Component
     }
     componentDidMount()
     {
-      fetch(`https://swapi.co/api/people/?search=r2`)
+      fetch(`http://192.168.2.128:5000/`)
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.js]: put these in some config file ... ! as links should be configurable

File: src/Components/Rectangle/index.js [hunk 13]
```diff
@@ -50,7 +50,7 @@ class Rectangle extends Component
         )
     }
     componentDid(id) {
-      fetch(`https://swapi.co/api/people/?search=${id}`)
+      fetch(`http://192.168.2.128:5000/search?name=${id}`)
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.js]: put these in some config file ... ! as links should be configurable

File: src/index.css [hunk 14]
```diff
@@ -34,122 +34,128 @@
...
           margin-left:3.14%;
-          margin-top:6px; 
-          font-size: 19px;
-          /* font-weight: 600; */
+          margin-top:10px; 
+          font-size: 16px;
           font-family: 'Muli', sans-serif;
         }
 
   .Div2 {
```
Comment (Deepankar01) [inline src/index.css]: lower-case only

File: src/index.css [hunk 15]
```diff
@@ -34,122 +34,128 @@
...
+          .Search
+        {
+            width:32%;
+            height: 100%;
+            margin-left:2%;
+          
+              
+        }
 
   .Div3 {
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 16]
```diff
@@ -34,122 +34,128 @@
...
-  }
-  .Search
-{
-    width:32%;
-    height: 100%;
-    margin-left:2%;
-  
-       
-}
+          .Tag {
```
Comment (Deepankar01) [inline src/index.css]: shouldn't this be in some component in Tag

File: src/index.css [hunk 17]
```diff
@@ -34,122 +34,128 @@
...
     grid-template-columns: 32% 32% 32%;
     
   }
 
-@media (min-width: 550px) and (max-width: 767px) {
+
+
+@media (min-width: 481px) and (max-width: 767px) {
+  
   .Div1 {
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 18]
```diff
@@ -34,122 +34,128 @@
...
           font-size: 25px;
         }
 
         .Coming-together-is-a{
-          font-size: 12px;
+          font-size: 14px;
+          width: 90%;
         }
 
   .Div2 {
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 19]
```diff
@@ -34,122 +34,128 @@
...
-    height: 4vh;
+    height: 32px;
   }
   .Div3{
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 20]
```diff
@@ -34,122 +34,128 @@
...
   
 }
-@media (max-width: 550px) 
-{
+
+
+
+@media (max-width: 480px) {
+  
   .Div1 {
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 21]
```diff
@@ -34,122 +34,128 @@
...
-          font-size: 19px;
+          font-size: 18px;
         }
 
         .Coming-together-is-a{
           font-size: 11px;
+          width: 90%;
         }
 
   .Div2 {
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 22]
```diff
@@ -34,122 +34,128 @@
...
         {
           height: 50%;
           width: 100%;
           background-color: blue; 
-          margin-top: 1.3%; 
+          margin-top: 2%; 
           margin-left:0%;
         }
 
   .Div3{
```
Comment (Deepankar01) [inline src/index.css]: lower case only

File: src/index.css [hunk 23]
```diff
@@ -34,122 +34,128 @@
...
-    grid-row-gap: 3%;
+    grid-row-gap: 1%;
     grid-template-columns: 100%;
   }
-  
- 
+
   
 }
 @import url('https://fonts.googleapis.com/css?family=Muli:400');
```
Comment (Deepankar01) [inline src/index.css]: this is the correct place to call this but since you have imported again in one of the files above please remove it from there

**Review by Deepankar01 — CHANGES_REQUESTED** [review]
Please fix the changes then will merge it

File: src/Components/Box/index.css [hunk 24]
```diff
@@ -40,10 +40,10 @@
         /* background-color:burlywood; */
       }
 
-.Rectangle-2:hover
+.Rectangle-2:focus, .Rectangle-2:hover
```
Comment (Deepankar01) [inline src/Components/Box/index.css]: The name is improper write proper name for it

File: src/Components/Box/index.js [hunk 25]
```diff
@@ -9,7 +9,7 @@ export default class Box extends Component {
 
     render() {
       return (
-        <div className="Rectangle-2">
+        <div className="Rectangle-2" onClick="return true">
```
Comment (Deepankar01) [inline src/Components/Box/index.js]: same here

File: src/Components/Rectangle/index.css [hunk 26]
```diff
@@ -1,11 +1,23 @@
-.Loading {
+
+  .Div3 {
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.css]: lower case

File: src/Components/Rectangle/index.css [hunk 27]
```diff
@@ -27,11 +39,34 @@
...
+    .Div3{
+      margin-top: 2%;
+      display: grid;
+      grid-column-gap: 2%;
+      grid-row-gap: 3%;
+      grid-template-columns: 49% 49%;
+    }  
+  }
 
+<<<<<<< HEAD
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.css]: this is messed up

File: src/Components/Rectangle/index.css [hunk 28]
```diff
@@ -27,11 +39,34 @@
...
+ 
+    .Div3{
+      margin-top: 5%;
+      display: grid;
+      grid-column-gap: 2%;
+      grid-row-gap: 1%;
+      grid-template-columns: 100%;
+    } 
+  }
+=======
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.css]: this is messed up

File: src/Components/Rectangle/index.css [hunk 29]
```diff
@@ -27,11 +39,34 @@
...
 /* 
   @media (max-width: 550px) 
   {
   .contribution{
     width:70%;
   }
-} */
\ No newline at end of file
+} */
+>>>>>>> 29f9dcc7820d62896e1e0a27d8f89248002acb2b
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.css]: this is messed up

File: src/Components/Rectangle/index.js [hunk 30]
```diff
@@ -50,7 +50,11 @@ class Rectangle extends Component
         )
     }
     componentDid(id) {
+<<<<<<< HEAD
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.js]: this is messed up

File: src/Components/Rectangle/index.js [hunk 31]
```diff
@@ -50,7 +50,11 @@ class Rectangle extends Component
...
+      fetch(`http://192.168.2.128:5000/search?queryparam=${id}`)
+=======
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.js]: this is messed up

File: src/Components/Rectangle/index.js [hunk 32]
```diff
@@ -50,7 +50,11 @@ class Rectangle extends Component
...
       fetch(`http://192.168.2.128:5000/search?name=${id}`)
+>>>>>>> 29f9dcc7820d62896e1e0a27d8f89248002acb2b
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.js]: this is messed up

File: src/Components/Rectangle/index.js [hunk 33]
```diff
@@ -75,14 +79,8 @@ class Rectangle extends Component
...
-        <div className="Rectangle-3">
-          <div className="Div1" >
-            <div className="Wall-of-Contribution" >WALL OF CONTRIBUTION</div>
-            <div className="Coming-together-is-a">Coming together is a beginning. Keeping together is progress. Working together is success.</div>
-          </div>
-
+        <div>
           <Search onValueChange={this.onValChange} onGoClick={this.onGoClick} />
-
           <div className="Div3">
```
Comment (Deepankar01) [inline src/Components/Rectangle/index.js]: lower case class names

File: src/Components/Search/index.css [hunk 34]
```diff
@@ -1,48 +1,67 @@
...
+            letter-spacing: 0.4px;
+            color: #ffffff;
+          
+      } 
 
-  }
 
   @media (min-width: 481px) and (max-width: 767px) {
   
+    .Div2 {
```
Comment (Deepankar01) [inline src/Components/Search/index.css]: lowercase

**Review by Deepankar01 — CHANGES_REQUESTED** [review]
still not correct

File: public/index.html (hunk 2 above)
Comment (kritiagrawal1997) [reply public/index.html:13]: Sir, we needed bootstrap for alignment and tags

**Pull request closed without merging**